design database model

Gurnav224 opened the pull request:
design the database model for the application

created required model 
1. user 
2. media
3. message
4. room

kenf1 commented on apps/backend/src/models/user.model.ts:
```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+
+const userSchema = new mongoose.Schema({
```
is this model for auth or pass data into message.model.ts and room.model.ts?

Gurnav224 commented on apps/backend/src/models/room.model.ts:
```diff
@@ -0,0 +1,16 @@
+  hostUserId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+  mediaId: { type: mongoose.Schema.Types.ObjectId, ref: 'Media' },
+  playback: {
+    state: { type: String, enum: ['playing', 'paused'], default: 'paused' },
```
ok , I will the enum to global scope

Gurnav224 commented:
Hey @kenf1, I’ve added a model with more context. I also added a new doc: https://github.com/buildiverse/shared-media-streaming/pull/32/commits/88e2678d2ec0e3eb7073961c7b84ce7c79770ddf

It includes a detailed explanation of the database model and their relationships. Please check it once and let me know if anything needs to be changed.

kenf1 commented:
looks good
